ScoutSuite/providers/utils.py

```diff
@@ -2,13 +2,13 @@
 
 
 def get_non_provider_id(name):
-        """
-        Not all resources have an ID and some services allow the use of "." in names, which breaks Scout's
-        recursion scheme if name is used as an ID. Use SHA1(name) instead.
+    """
+    Not all resources have an ID and some services allow the use of "." in names, which breaks Scout's
+    recursion scheme if name is used as an ID. Use SHA1(name) instead.
 
-        :param name:                    Name of the resource to
-        :return:                        SHA1(name)
-        """
-        _hash = sha1()
-        _hash.update(name.encode('utf-8'))
-        return _hash.hexdigest()
+    :param name:                    Name of the resource to
+    :return:                        SHA1(name)
+    """
+    hash = sha1()
+    hash.update(name.encode('utf-8'))
+    return hash.hexdigest()
```

Comment:
No hate, the indentation was broken, and I think using _ or local variables is not PEP 8 compliant.

Reply:
`hash` is a python built-in.

Reply:
lol